src/auto_archiver/modules/instagram_api_extractor/__manifest__.py

```diff
@@ -31,9 +31,11 @@
         },
     },
     "description": """
-Archives various types of Instagram content using the Instagrapi API.
+Archives Instagram content using a deployment of the [Instagrapi API](https://subzeroid.github.io/instagrapi/).
 
-Requires setting up an Instagrapi API deployment and providing an access token and API endpoint.
+Requires either getting a token from using a hosted [(paid) service](https://api.instagrapi.com/docs) and setting this in the configuration file.
+Alternatively you can run your own server. We have a basic script which you can use for this which can be ran locally or using Docker.
+For more information, read the [how to guide](https://auto-archiver.readthedocs.io/en/latest/how_to/run_instagrapi_server.html) on this.
```

Comment:
ideally this shouldn't be hard-coded to the latest version. Should use:

`[how to guide](../../../run_instagrapi_server.md)` ( I think I got the reference right!)

Reply:
```suggestion
For more information, read the [how to guide](../../../how_to/run_instagrapi_server.md) on this.
```